src/libraries/CurrencyTeller.sol

```diff
@@ -1,13 +1,11 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.20;
 
-import {Currency, CurrencyLibrary} from "../types/Currency.sol";
+import {Currency} from "../types/Currency.sol";
 import {IERC20Minimal} from "../interfaces/external/IERC20Minimal.sol";
 import {IPoolManager} from "../interfaces/IPoolManager.sol";
 
-library CurrencySettleTake {
-    using CurrencyLibrary for Currency;
-
+library CurrencyTeller {
```

Comment:
this is maybe the most questionable change... i just dont like the name `CurrencySettleTake`... open to suggestions

Reply:
here's a list from chatgpt lol. I kind of like CurrencyFlow myself

```
CurrencyManager
CurrencyHandler
CurrencyOps
CurrencyService
CurrencyProcessor
CurrencyFlow
CurrencyController
```